### PR TITLE
feat(javascript): Better comments and instructions

### PR DESCRIPTION
**Pull request by seebees**
Make the comments and instructions more targeted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: exercises/node-javascript/add-esdk-complete/retrieve.js [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
- */
-
+// ADD-ESDK-START: Add the dependency
 const { decryptStream, KmsKeyringNode } = require("@aws-crypto/client-node");
 const { S3 } = require("aws-sdk");
 const s3 = new S3();
 const config = require("./config");
 const Bucket = config.state.bucketName();
 
+// ADD-ESDK-START: Add the dependency
```
Comment (agray256) [inline exercises/node-javascript/add-esdk-complete/retrieve.js]: // ADD-ESDK-START: Set up a keyring to use Faythe's CMK for decrypting.

File: exercises/node-javascript/add-esdk-complete/store.js [hunk 2]
```diff
@@ -1,13 +1,7 @@
...
 // SPDX-License-Identifier: Apache-2.0
 
-/* ADD-ESDK-START:
- * Add the @aws-crypto/client-node dependency.
- * You will need the encrypt stream as well as the KMS keyring.
- * Look at the ./config.js file to see how to pull the Faythe CMK.
- * Then pipe the data stream to the encrypt stream: Profit.
- */
-
+// ADD-ESDK-START: Add the dependency
```
Comment (agray256) [inline exercises/node-javascript/add-esdk-complete/store.js]: // ADD-ESDK-START: Add the @aws-crypto/client-node dependency

File: exercises/node-javascript/add-esdk-start/retrieve.js [hunk 3]
```diff
@@ -1,20 +1,18 @@
...
- * Then pipe the encrypted stream to the decrypt stream: Profit.
- */
+// ADD-ESDK-START: Add the dependency
 
 const { S3 } = require("aws-sdk");
 const s3 = new S3();
 const config = require("./config");
 const Bucket = config.state.bucketName();
 
+// ADD-ESDK-START: Plumb In Your Config
```
Comment (agray256) [inline exercises/node-javascript/add-esdk-start/retrieve.js]: // ADD-ESDK-START: Set up a keyring to use Faythe's CMK for decrypting.

File: exercises/node-javascript/add-esdk-complete/retrieve.js (hunk 1 above)
Comment (seebees) [reply exercises/node-javascript/add-esdk-complete/retrieve.js]: // ADD-ESDK-START: Plumb In Your Config

That matches the instructions.  Pushed.

File: exercises/node-javascript/add-esdk-start/retrieve.js (hunk 3 above)
Comment (seebees) [reply exercises/node-javascript/add-esdk-start/retrieve.js]: @lizroth What do you think of these comments?  I like them more, but the titles of the various instructions should match.

Thoughts?

Comment (lizroth) [reply exercises/node-javascript/add-esdk-start/retrieve.js]: Strong agree, I was planning to take a crack at them tomorrow with a slightly fresher perspective, but I think we should do them.

https://github.com/aws-samples/busy-engineers-document-bucket/issues/66

Comment (seebees) [reply exercises/node-javascript/add-esdk-start/retrieve.js]: KK, I will update both the instructions and these comments

File: exercises/node-javascript/encryption-context-start/store.js [hunk 4]
```diff
@@ -45,31 +45,15 @@ async function store(fileStream, encryptionContext = {}) {
...
-     * This can create hot keys when searching for a given ec key...
-     * Better to write shard this out `${hash(Key)}#${contextKey}`
-     * Where `hash(Key)` will return 0-n where n is the number of shards
-     */
     .map(canonicalKey => ddbItem(canonicalKey, Key));
 
-  /* ENCRYPTION-CONTEXT-START:
-   * The included `encryptionContext` in the encrypt call option hash.
-   */
+  // ENCRYPTION-CONTEXT-START: The included `encryptionContext` in the encrypt call option hash.
```
Comment (WesleyRosenblum) [inline exercises/node-javascript/encryption-context-start/store.js]: This isn't a complete sentence, can it be made more clear what is expected?

**Pull request merged**